plugin-server/src/worker/ingestion/team-manager.ts

```diff
@@ -279,6 +281,11 @@ DO UPDATE SET property_type=$5, property_type_format=$6 WHERE posthog_propertyde
                     'insertPropertyDefinition'
                 )
                 this.propertyDefinitionsCache.get(team.id)?.set(key, propertyType || NULL_AFTER_PROPERTY_TYPE_DETECTION)
+
+                this.statsd?.gauge(
+                    `propertyDefinitionsCacheLength.team${team.id}`,
```

Comment:
Might be worth to add a space or a dot or something to visually separate "team" and "123"?